### PR TITLE
Want nfs

### PR DESCRIPTION
**Pull request by ZeroChaos-**

### CODE REVIEW

File: init.d/netmount.in [hunk 1]
```diff
@@ -6,8 +6,13 @@ description="Mounts network shares according to /etc/fstab."
 
 depend()
 {
-	config /etc/fstab
-	use afc-client amd nfsclient autofs openvpn
+	local opts mywant=""
```
Comment (dwfreed) [inline init.d/netmount.in]: This defines an unused local variable 'opts'.

Comment (ZeroChaos-) [inline init.d/netmount.in]: and now it's not unused :-)

**Comment by ZeroChaos-** [comment]
yeah, I wasn't sure about how I wanted to deal with that but it seemed safest to have the if fstabinfo bit.  just in case openrc chooses to start outputting some message and not just exit

**Comment by ZeroChaos-** [comment]
tested and confirmed, this works properly.

**Pull request closed without merging**